### PR TITLE
fixed navbar bug and chat card bug

### PR DESCRIPTION
**Pull request by Fermec28**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -10,25 +10,16 @@
...
-//side bar functionality
-(function($){
-  $(function(){
-
-    $('.sidenav').sidenav();
-    $("#cart-container").on("click","li",function(e){ $(this).find( "a" )[0].click();})
-  }); // end of document ready
-})(jQuery);
\ No newline at end of file
+$(document).ready(function(){
```
Comment (bogota-bootcamp) [inline app/assets/javascripts/application.js:23]: why is this empty?
?porque es nada aqui?

File: app/assets/javascripts/conversations.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 /*Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/ */
-$("#cart-container").on("click","li",function(e){ $(this).find( "a" )[0].click();})
\ No newline at end of file
+$(document).ready(function(){
+
```
Comment (bogota-bootcamp) [inline app/assets/javascripts/conversations.js:5]: again why is this blank?
aqui tambien, porque nada?

**Review by bogota-bootcamp — APPROVED** [review]
a couple of empty document.ready blocks but it looks fine

File: app/assets/javascripts/conversations.js (hunk 2 above)
Comment (Fermec28) [reply app/assets/javascripts/conversations.js:5]: No se recargaba el código cuando cambiaba la pagina, lo inserte directo en las vistas.
Beacuse the js was not reloaded when the page changes, then the events were not defined again.

**Pull request merged**